source/articles/sql/Bulk-Copy.md

```diff
@@ -44,19 +53,21 @@ SQLite               |   Yes    |     Yes      |        No        | MultipleRows
+
 ### PostgreSQL provider-specific bulk copy
 
-For PostgreSQL `BulkCopy` use `BINARY COPY` operation when `ProviderSpecific` method specified. This operation is very sensitive to what types are used. You must always use proper type that match type in target table, or you will receive error from server (e.g. `"22P03: incorrect binary data format"`).
+For PostgreSQL, `BulkCopy` uses the `BINARY COPY` operation when the `ProviderSpecific` method specified. This operation is very sensitive to what types are used. You must always use the proper type that matches the type in target table, or you will receive an error from server (e.g. `"22P03: incorrect binary data format"`).
 
-Below is a list of types, that could result in error without explicit type specification:
+Below is a list of types that could result in error without an explicit type specification:
 
-- `decimal`/`numeric` vs `money`. Those are two different types, mapped to `System.Decimal`. Default mappings will use `numeric` type, so if your column has `money` type, you should type it in mapping using `DataType = DataType.Money` or `DbType = "money"` hints.
-- `time` vs `interval`. Those are two different types, mapped to `System.TimeSpan`. Default mappings will use `time` type, so if your column has `interval` type, you should type it in mapping using `DbType = "interval"` hint. Or use `NpgsqlTimeSpan` type for intervals.
-- any text types/`json` vs `jsonb`. All those types mapped to `System.String` (except `character` which is mapped to `System.Char`). Default mappings will not work for `jsonb` column and you should type it in mapping using `DataType = DataType.BinaryJson` or `DbType = "jsonb"` hint.
-- `inet` vs `cidr`. If you use `NpgsqlInet` type for mapping column, it could be mapped to both `inet` and 'cidr' types. There is no default mapping for this type, so you should explicitly specify it using `DbType = "inet"` or `DbType = "cidr"` hint. Also for `inet` you can use `IPAddress` which will be mapped to `inet` type.
-- `macaddr` vs `macaddr8`. Both types could be mapped to the same `PhysicalAddress`/`String` types, so you should explicitly specify column type using `DbType = "macaddr"` or `DbType = "macaddr8"` hints. Even if you use provider version without `macaddr8` support, you should specify hint or it will break after provider update to newer version.
-- `date` type. You should use `NpgsqlDate` type in mapping or specify `DataType = DataType.Date` or `DbType = "date"` hints.
-- `time with time zone` type needs `DbType = "time with time zone"` hint.
+- `decimal`/`numeric` vs `money`. Those are two different types, mapped to `System.Decimal`. Default mappings will use `numeric` type, so if your column is the `money` type, you should type it in mappings using `DataType = DataType.Money` or `DbType = "money"` hints.
```

Comment:
`Default mappings will use `numeric` type `

to

`Default mappings will use the `numeric` type`

Reply:
Done